### PR TITLE
Docstring parse errors and switch from 'string' to 'str'

### PR DESCRIPTION
**Pull request by daspecster**
- There were just a couple typos with a semi-colon instead of a colon.
- Also, in the files that I had touched I converted types from `string` to `str` which is better supported by the documentation parsers.

### CODE REVIEW

File: gcloud/monitoring/query.py [hunk 1]
```diff
@@ -219,6 +219,7 @@ def select_projects(self, *args):
             query = query.select_projects('project-1')
             query = query.select_projects('project-1', 'project-2')
 
+        :type args: str
```
Comment (dhermes) [inline gcloud/monitoring/query.py]: `*args` is always a tuple.

File: gcloud/monitoring/query.py [hunk 2]
```diff
@@ -268,10 +269,12 @@ def select_resources(self, *args, **kwargs):
             not a resource label. You would filter on it using
             ``select_metrics(instance_name=...)``.
 
+        :type args: str
```
Comment (dhermes) [inline gcloud/monitoring/query.py]: Ditto

File: gcloud/monitoring/query.py [hunk 3]
```diff
@@ -268,10 +269,12 @@ def select_resources(self, *args, **kwargs):
...
         :param args: Raw filter expression strings to include in the
             conjunction. If just one is provided and no keyword arguments
             are provided, it can be a disjunction.
 
+        :type kwargs: str
```
Comment (dhermes) [inline gcloud/monitoring/query.py]: Dict

File: gcloud/monitoring/timeseries.py [hunk 4]
```diff
@@ -127,6 +127,7 @@ class Point(collections.namedtuple('Point', 'end_time start_time value')):
     :type start_time: string or None
     :param start_time: An optional start time in RFC3339 UTC "Zulu" format.
 
+    :type value: int
```
Comment (dhermes) [inline gcloud/monitoring/timeseries.py]: From the description as "scalar" I'd imagine it's a float. Any clue from the unit tests?

**Comment by dhermes** [comment]
Nice finds. I'm surprised `sphinx-docs` doesn't complain about some of those `;` for `:` typos

File: gcloud/monitoring/timeseries.py (hunk 4 above)
Comment (rimey) [inline gcloud/monitoring/timeseries.py]: A scalar, here, means an int, float, bool, or string. A distribution would be represented as a dictionary.

The precise details will evolve.

Comment (rimey) [inline gcloud/monitoring/timeseries.py]: Don't add the type annotation.

Comment (dhermes) [inline gcloud/monitoring/timeseries.py]: In `gcloud-python`, we have type annotations for all values (except when they fall through the cracks like this). A note like "any" or being literal and saying `object` or just giving the list of possible types @rimey just listed should all be fine.

Comment (daspecster) [inline gcloud/monitoring/timeseries.py]: pdoc fails if there isn't a type defined for a param, which is how I found it.

I'm game for an `object` for now. Thanks!

**Comment by daspecster** [comment]
I'm surprised too actually. I did some looking around briefly but I didn't noticed anything about sphinx overlooking those but my google foo probably wasn't very good.

Next push I put up for the `json-docs` branch I want to have a lint mode for this to run before actually outputting to files to help find things like this.

File: gcloud/monitoring/query.py [hunk 5]
```diff
@@ -309,10 +312,12 @@ def select_metrics(self, *args, **kwargs):
 
             metric.label.<label> = ends_with("<value>")
 
+        :type args: str
```
Comment (rimey) [inline gcloud/monitoring/query.py]: tuple

File: gcloud/streaming/http_wrapper.py [hunk 6]
```diff
@@ -434,6 +434,7 @@ def _register_http_factory(factory):
 def get_http(**kwds):
     """Construct an Http instance.
 
+    :type kwds: str, int
```
Comment (rimey) [inline gcloud/streaming/http_wrapper.py]: dict

**Comment by dhermes** [comment]
@daspecster Is this ready to go?

**Comment by daspecster** [comment]
If you guys think so, I'm good to go on it.

**Comment by dhermes** [comment]
LGTM. LMK if you can't merge.

**Pull request merged**